xml/art_migration.xml

```diff
@@ -0,0 +1,692 @@
+     </para>
+     </listitem>
+    </itemizedlist>
+   </listitem>
+   <listitem>
+    <para>
+     Copy the <filename>home</filename> directory from the seed to the new deployer:
+    </para>
+    <screen>(seed)> rsync -avP ~ <replaceable>NEW_DEPLOYER_IP</replaceable>:/home</screen>
+   </listitem>
+   <listitem>
+    <para>
+     Unpack or copy HOS 5.SCP contents to the <filename>home</filename> directory of the new
+     deployer.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Initialize the deployer node:
+    </para>
+    <screen>(new deployer)> cd ~/hos-5.1.0; ./hos-init.bash</screen>
+   </listitem>
+  </itemizedlist>
+ </step>
+</procedure>
+</sect1>
+<sect1>
+ <title>Migration from HOS 5.SCP to Ardana.Newton</title>
+ <para>
+  The following procedure describes the process of migrating from HOS 5.SCP to Ardana.Newton.
+ </para>
+  <procedure>
+   <title>Preparation Phase</title>
+   <step>
+    <para>
+     Convert the <filename>helion</filename> Git repo to <filename>openstack</filename> Git repo for use by Ardana:
+    </para>
+    <screen>> cd ~
+> cp -r helion openstack
+> cd openstack
+> git branch -m hos ardana
+> git checkout ardana
+> sed -i 's/hos/ardana/' .gitignore
+> git commit -a -m "Prep for Ardana"
+> git mv hos ardana
+> git mv hos_extensions ardana_extensions
+> git commit -a -m "Conversion to Ardana"
+> git checkout cp-persistent
+> sed -i 's/component: helion-ca/component: ardana-ca/' my_cloud/persistent_state/private_data_control-plane-1.yml
+> sed -i 's/helion_internal_ca/ardana_internal_ca/' my_cloud/persistent_state/private_data_control-plane-1.yml
+> git commit -a -m "Conversion to Ardana"
+> git checkout site
+# undo any manual changes that have been committed to the hos/ansible directory
+> git checkout hos -- hos/ansible   # 'hos' has been changed to ardana(line 9)? If so, 'hos/ansible' might be changed to 'ardana/ansible'(line 13)
+# if any files were modified, review their changes and note that the customizations will need to be reapplied in the ardana playbooks
+> git diff
+> git commit -a -m "Resetting codebase to Ardana"
+# finally, pull forward the changes made earlier on the ardana branch
+> git merge ardana -m "Conversion to Ardana"</screen>
+   </step>
+   <step>
+    <para>
+     Rename the old <filename>scratch</filename> directory to archive any remaining
+     HOS 5.SCP files:
+    </para>
+    <screen>mv ~/scratch ~/scratch.scp</screen>
+   </step>
+   <step>
+    <para>
+     Prepare the <filename>ardana_packager</filename> working directory to include existing &suse; repositories:
+    </para>
+    <screen>> sudo mkdir /opt/ardana_packager
+> sudo ln -s /opt/hlm_packager/hlm /opt/ardana_packager/hlm</screen>
+   </step>
+   <step>
+    <para>
+     Install Ardana.Newton content onto deployer:
+    </para>
+    <screen># untar ardana installer into ~/ardana-0.35.0
+> cd ~/ardana-0.35.0
+> ./ardana-init.bash</screen>
+    <para>
```

Comment:
Maybe make the rest a separate procedure and link to it..? It seems like it would fit better under a Troubleshooting headline.